### PR TITLE
New icons for generic payment methods

### PR DESCRIPTION
**Pull request by adeniyiao**
## Why are you adding this icons?
- Update existing generic icons for cash, generic  and gift card 
- Adding new generic payment method for wallet, online banking and store credit

#### Help us identify yourself
- [ ] I'm working/collaborating with the brand directly and they have provided the icons.
- [ ] I'm associated with the brand and I've read all the brand icon’s guidelines.
- [ ] I'm an individual and I've read all the brand icon’s guidelines.

Link to the brand guidelines: <Link>

## Checklist to add new icons
- [ ] All icons have a corresponding entry in `db/payment_icons.yml`
- [ ] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [ ] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [ ] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [ ] I have attached a screenshot comparison with the example icon provided in guidelines
- [ ] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

#### Tips how to create a screenshot
We have found free online SVG editor https://www.freecodeformat.com/svg-editor.php very useful to create one. Here is a sample code for you to verify that you icon appears properly along side the placeholder.

```
<!-- Change background color if needed to showcase your icon better -->
<style> body { background: black; } </style>

<!-- DO NOT DELETE EXAMPLE -->
<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" role="img" width="38" height="24" aria-labelledby="pi-visa"><title id="pi-visa">Visa</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path d="M28.3 10.1H28c-.4 1-.7 1.5-1 3h1.9c-.3-1.5-.3-2.2-.6-3zm2.9 5.9h-1.7c-.1 0-.1 0-.2-.1l-.2-.9-.1-.2h-2.4c-.1 0-.2 0-.2.2l-.3.9c0 .1-.1.1-.1.1h-2.1l.2-.5L27 8.7c0-.5.3-.7.8-.7h1.5c.1 0 .2 0 .2.2l1.4 6.5c.1.4.2.7.2 1.1.1.1.1.1.1.2zm-13.4-.3l.4-1.8c.1 0 .2.1.2.1.7.3 1.4.5 2.1.4.2 0 .5-.1.7-.2.5-.2.5-.7.1-1.1-.2-.2-.5-.3-.8-.5-.4-.2-.8-.4-1.1-.7-1.2-1-.8-2.4-.1-3.1.6-.4.9-.8 1.7-.8 1.2 0 2.5 0 3.1.2h.1c-.1.6-.2 1.1-.4 1.7-.5-.2-1-.4-1.5-.4-.3 0-.6 0-.9.1-.2 0-.3.1-.4.2-.2.2-.2.5 0 .7l.5.4c.4.2.8.4 1.1.6.5.3 1 .8 1.1 1.4.2.9-.1 1.7-.9 2.3-.5.4-.7.6-1.4.6-1.4 0-2.5.1-3.4-.2-.1.2-.1.2-.2.1zm-3.5.3c.1-.7.1-.7.2-1 .5-2.2 1-4.5 1.4-6.7.1-.2.1-.3.3-.3H18c-.2 1.2-.4 2.1-.7 3.2-.3 1.5-.6 3-1 4.5 0 .2-.1.2-.3.2M5 8.2c0-.1.2-.2.3-.2h3.4c.5 0 .9.3 1 .8l.9 4.4c0 .1 0 .1.1.2 0-.1.1-.1.1-.1l2.1-5.1c-.1-.1 0-.2.1-.2h2.1c0 .1 0 .1-.1.2l-3.1 7.3c-.1.2-.1.3-.2.4-.1.1-.3 0-.5 0H9.7c-.1 0-.2 0-.2-.2L7.9 9.5c-.2-.2-.5-.5-.9-.6-.6-.3-1.7-.5-1.9-.5L5 8.2z" fill="#142688"/></svg>

<!-- TODO: insert your icon here -->
<YOUR SVG CODE>

<br>
<!-- TODO: insert your icon here -->
<YOUR SVG CODE>
</br
```

#### If the icons are intended for use by Shopify, please provide the following info:
Who are you working with at Shopify? (avoid adding personal details, provide github handle(preferred) or first name and last name)

What's the expected date of this change to deploy on Shopify?

### CODE REVIEW

File: app/assets/images/payment_icons/cash.svg [hunk 1]
```diff
@@ -1 +1,2 @@
-<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" aria-labelledby="pi-cash" width="38" height="24" role="img"><title id="pi-cash">Cash</title><path fill="#98C74C" d="M38 21a3 3 0 0 1-3 3H3a3 3 0 0 1-3-3V3a3 3 0 0 1 3-3h32a3 3 0 0 1 3 3v18z"/><circle opacity=".2" cx="19" cy="12" r="8"/><circle opacity=".2" cx="32" cy="12" r="2"/><circle opacity=".2" cx="6" cy="12" r="2"/><path fill="#fff" d="M22.113 13.897c0-1.573-1.271-2.067-2.518-2.42V8.763c.65.118 1.218.588 1.68 1.105l.71-.797c-.578-.609-1.424-1.128-2.39-1.246V6.83h-1.034v.973c-1.408.154-2.413 1.036-2.413 2.275 0 1.461 1.2 1.907 2.413 2.244v3.024c-.898-.145-1.608-.759-2.053-1.277l-.62.945c.597.673 1.504 1.151 2.674 1.266v.89h1.034v-.881c1.819-.186 2.517-1.304 2.517-2.392zm-4.974-3.948c0-.684.627-1.11 1.422-1.223v2.459c-.797-.133-1.422-.596-1.422-1.236zm2.456 5.426v-2.762c.813.254 1.451.59 1.451 1.359 0 .565-.364 1.246-1.451 1.403z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-{cash}">
```
Comment (jffortier) [inline app/assets/images/payment_icons/cash.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-cash">
```

This needs to match the `id` set on the `<title>` tag to create the right connection.

File: app/assets/images/payment_icons/cash.svg [hunk 2]
```diff
@@ -1 +1,2 @@
...
...1.617-1.41ZM21.587 14.161c0 .737-.518 1.328-1.837 1.475v-2.982c1.437.325 1.837.792 1.837 1.507Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/><path fill-rule="evenodd" clip-rule="evenodd" d="M19 21a9 9 0 1 0 0-18 9 9 0 0 0 0 18Zm.002-3H19a.75.75 0 0 1-.75-.75v-.561c-1.542-.176-2.84-.871-3.201-2.01a1.005 1.005 0 0 1-.049-.307c0-.327.276-.551.666-.551.333 0 .585.143.723.483.248.649.795 1.152 1.861 1.315v-3.271c-1.971-.412-3.03-1.166-3.03-2.489 0-1.395 1.246-2.338 3.03-2.547V6.75A.75.75 0 0 1 19 6h.004a.75.75 0 0 1 .746.75v.555c1.495.18 2.677.915 3.03 2.01.033.102.05.204.05.306 0 .327-.277.551-.666.551-.382 0-.61-.19-.723-.483-.325-.717-.87-1.147-1.691-1.302v3.107c2.083.422 3.25 1.144 3.25 2.572 0 1.566-1.45 2.45-3.25 2.633v.551a.75.75 0 0 1-.746.75h-.002Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/><path d="M5 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4ZM33 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/></g><defs><clipPath id="a"><path fill="#fff" style="fill:#fff;fill-opacity:1" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```
Comment (jffortier) [inline app/assets/images/payment_icons/cash.svg:2]:
```suggestion
    <title id="pi-cash">Cash</title><g clip-path="url(#a)"><rect width="38" height="24" rx="3" fill="#F5F5F5" /><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" /><path d="M16.633 9.778c0-.611.556-1.193 1.617-1.384v2.794c-1.193-.296-1.617-.788-1.617-1.41ZM21.587 14.161c0 .737-.518 1.328-1.837 1.475v-2.982c1.437.325 1.837.792 1.837 1.507Z" fill="#707070" /><path fill-rule="evenodd" clip-rule="evenodd" d="M19 21a9 9 0 1 0 0-18 9 9 0 0 0 0 18Zm.002-3H19a.75.75 0 0 1-.75-.75v-.561c-1.542-.176-2.84-.871-3.201-2.01a1.005 1.005 0 0 1-.049-.307c0-.327.276-.551.666-.551.333 0 .585.143.723.483.248.649.795 1.152 1.861 1.315v-3.271c-1.971-.412-3.03-1.166-3.03-2.489 0-1.395 1.246-2.338 3.03-2.547V6.75A.75.75 0 0 1 19 6h.004a.75.75 0 0 1 .746.75v.555c1.495.18 2.677.915 3.03 2.01.033.102.05.204.05.306 0 .327-.277.551-.666.551-.382 0-.61-.19-.723-.483-.325-.717-.87-1.147-1.691-1.302v3.107c2.083.422 3.25 1.144 3.25 2.572 0 1.566-1.45 2.45-3.25 2.633v.551a.75.75 0 0 1-.746.75h-.002Z" fill="#707070" /><path d="M5 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4ZM33 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4Z" fill="#707070" /></g><defs><clipPath id="a"><path fill="#fff" style="fill:#fff;fill-opacity:1" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```

The `style` tag is redundant and p3 colors are not supported by all browsers. The `fill` is enough.

Comment (jffortier) [reply app/assets/images/payment_icons/cash.svg:2]: Is the stoke opacity intentional?

File: app/assets/images/payment_icons/generic.svg [hunk 3]
```diff
@@ -1 +1,2 @@
-<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" aria-labelledby="pi-generic" width="38" height="24" role="img"><title id="pi-generic">Generic</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path fill="#999" d="M0 5h38v4H0V5z"/><path opacity=".15" d="M5 17h10v2H5v-2zm24 0h4v2h-4v-2z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-{generic}">
```
Comment (jffortier) [inline app/assets/images/payment_icons/generic.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-generic">
```

File: app/assets/images/payment_icons/generic.svg [hunk 4]
```diff
@@ -1 +1,2 @@
...
+    <title id="pi-generic">Generic</title><g clip-path="url(#a)"><rect width="38" height="24" rx="3" fill="#F5F5F5" style="fill:#f5f5f5;fill:color(display-p3 .9608 .9608 .9608);fill-opacity:1"/><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" style="stroke:#000;stroke-opacity:.07"/><path d="M37 4H1v3h36V4ZM4 19a1 1 0 0 1 1-1h9a1 1 0 1 1 0 2H5a1 1 0 0 1-1-1ZM29 19a1 1 0 0 1 1-1h3a1 1 0 1 1 0 2h-3a1 1 0 0 1-1-1Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/></g><defs><clipPath id="a"><path fill="#fff" style="fill:#fff;fill-opacity:1" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```
Comment (jffortier) [inline app/assets/images/payment_icons/generic.svg:2]:
```suggestion
    <title id="pi-generic">Generic</title><g clip-path="url(#a)"><rect width="38" height="24" rx="3" fill="#F5F5F5" /><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" style="stroke:#000;stroke-opacity:.07"/><path d="M37 4H1v3h36V4ZM4 19a1 1 0 0 1 1-1h9a1 1 0 1 1 0 2H5a1 1 0 0 1-1-1ZM29 19a1 1 0 0 1 1-1h3a1 1 0 1 1 0 2h-3a1 1 0 0 1-1-1Z" fill="#707070" /></g><defs><clipPath id="a"><path fill="#fff" /></clipPath></defs></svg>
```

**Comment by adeniyiao** [comment]
Thanks @jffortier for the review, I saw your comments late before making some edits. Could you please review again based on latest changes

**Review by Jacquesattaque — APPROVED** [review]

File: app/assets/images/payment_icons/cash.svg [hunk 5]
```diff
@@ -1 +1 @@
...
...1.41ZM21.587 14.161c0 .737-.518 1.328-1.837 1.475v-2.982c1.437.325 1.837.792 1.837 1.507Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/><path fill-rule="evenodd" clip-rule="evenodd" d="M19 21a9 9 0 1 0 0-18 9 9 0 0 0 0 18Zm.002-3H19a.75.75 0 0 1-.75-.75v-.561c-1.542-.176-2.84-.871-3.201-2.01a1.005 1.005 0 0 1-.049-.307c0-.327.276-.551.666-.551.333 0 .585.143.723.483.248.649.795 1.152 1.861 1.315v-3.271c-1.971-.412-3.03-1.166-3.03-2.489 0-1.395 1.246-2.338 3.03-2.547V6.75A.75.75 0 0 1 19 6h.004a.75.75 0 0 1 .746.75v.555c1.495.18 2.677.915 3.03 2.01.033.102.05.204.05.306 0 .327-.277.551-.666.551-.382 0-.61-.19-.723-.483-.325-.717-.87-1.147-1.691-1.302v3.107c2.083.422 3.25 1.144 3.25 2.572 0 1.566-1.45 2.45-3.25 2.633v.551a.75.75 0 0 1-.746.75h-.002Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/><path d="M5 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4ZM33 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/></g><defs><clipPath id="pi-cash"><path fill="#fff" style="fill:#fff;fill-opacity:1" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```
Comment (jffortier) [inline app/assets/images/payment_icons/cash.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-cash"><title id="pi-cash">Cash</title><g clip-path="url(#pi-cash)"><rect width="38" height="24" rx="3" fill="#F5F5F5" /><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" /><path d="M16.633 9.778c0-.611.556-1.193 1.617-1.384v2.794c-1.193-.296-1.617-.788-1.617-1.41ZM21.587 14.161c0 .737-.518 1.328-1.837 1.475v-2.982c1.437.325 1.837.792 1.837 1.507Z" fill="#707070" /><path fill-rule="evenodd" clip-rule="evenodd" d="M19 21a9 9 0 1 0 0-18 9 9 0 0 0 0 18Zm.002-3H19a.75.75 0 0 1-.75-.75v-.561c-1.542-.176-2.84-.871-3.201-2.01a1.005 1.005 0 0 1-.049-.307c0-.327.276-.551.666-.551.333 0 .585.143.723.483.248.649.795 1.152 1.861 1.315v-3.271c-1.971-.412-3.03-1.166-3.03-2.489 0-1.395 1.246-2.338 3.03-2.547V6.75A.75.75 0 0 1 19 6h.004a.75.75 0 0 1 .746.75v.555c1.495.18 2.677.915 3.03 2.01.033.102.05.204.05.306 0 .327-.277.551-.666.551-.382 0-.61-.19-.723-.483-.325-.717-.87-1.147-1.691-1.302v3.107c2.083.422 3.25 1.144 3.25 2.572 0 1.566-1.45 2.45-3.25 2.633v.551a.75.75 0 0 1-.746.75h-.002Z" fill="#707070" /><path d="M5 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4ZM33 14a2 2 0 1 0 0-4 2 2 0 0 0 0 4Z" fill="#707070" /></g><defs><clipPath id="pi-cash"><path fill="#fff"  d="M0 0h38v24H0z"/></clipPath></defs></svg>
```

File: app/assets/images/payment_icons/generic.svg [hunk 6]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-generic"><title id="pi-generic">Generic</title><g clip-path="url(#pi-generic)"><rect width="38" height="24" rx="3" fill="#F5F5F5" style="fill:#f5f5f5;fill:color(display-p3 .9608 .9608 .9608);fill-opacity:1"/><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" style="stroke:#000;stroke-opacity:.07"/><path d="M37 4H1v3h36V4ZM4 19a1 1 0 0 1 1-1h9a1 1 0 1 1 0 2H5a1 1 0 0 1-1-1ZM29 19a1 1 0 0 1 1-1h3a1 1 0 1 1 0 2h-3a1 1 0 0 1-1-1Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/></g><defs><clipPath id="pi-generic"><path fill="#fff" style="fill:#fff;fill-opacity:1" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```
Comment (jffortier) [inline app/assets/images/payment_icons/generic.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-generic"><title id="pi-generic">Generic</title><g clip-path="url(#pi-generic)"><rect width="38" height="24" rx="3" fill="#F5F5F5" /><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" /><path d="M37 4H1v3h36V4ZM4 19a1 1 0 0 1 1-1h9a1 1 0 1 1 0 2H5a1 1 0 0 1-1-1ZM29 19a1 1 0 0 1 1-1h3a1 1 0 1 1 0 2h-3a1 1 0 0 1-1-1Z" fill="#707070" /></g><defs><clipPath id="pi-generic"><path fill="#fff" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```

File: app/assets/images/payment_icons/gift-card.svg [hunk 7]
```diff
@@ -1 +1 @@
...
...//www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-gift-card"><title id="pi-gift-card">Gift Card</title><g clip-path="url(#pi-gift-card)"><rect width="38" height="24" rx="3" fill="#F5F5F5" style="fill:#f5f5f5;fill:color(display-p3 .9608 .9608 .9608);fill-opacity:1"/><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" style="stroke:#000;stroke-opacity:.07"/><path fill-rule="evenodd" clip-rule="evenodd" d="M15.275 12c.939 1.246 2.296 2 3.725 2a1 1 0 1 1 0 2c-2.002 0-3.75-.932-5-2.397V24h-2V13.603C10.75 15.068 9.002 16 7 16a1 1 0 1 1 0-2c1.43 0 2.786-.754 3.725-2H0v-2h5.385A3.129 3.129 0 0 1 5 8.5C5 6.567 6.79 5 9 5c1.195 0 2.267.458 3 1.185V0h2v6.185C14.733 5.458 15.805 5 17 5c2.21 0 4 1.567 4 3.5a3.13 3.13 0 0 1-.385 1.5H38v2H15.275ZM17 10c1.364 0 2-.914 2-1.5S18.364 7 17 7c-1.364 0-2 .914-2 1.5V10h2Zm-6 0V8.5c0-.586-.636-1.5-2-1.5-1.364 0-2 .914-2 1.5S7.636 10 9 10h2Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/></g><defs><clipPath id="pi-gift-card"><path fill="#fff" style="fill:#fff;fill-opacity:1" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```
Comment (jffortier) [inline app/assets/images/payment_icons/gift-card.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-gift-card"><title id="pi-gift-card">Gift Card</title><g clip-path="url(#pi-gift-card)"><rect width="38" height="24" rx="3" fill="#F5F5F5" /><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" /><path fill-rule="evenodd" clip-rule="evenodd" d="M15.275 12c.939 1.246 2.296 2 3.725 2a1 1 0 1 1 0 2c-2.002 0-3.75-.932-5-2.397V24h-2V13.603C10.75 15.068 9.002 16 7 16a1 1 0 1 1 0-2c1.43 0 2.786-.754 3.725-2H0v-2h5.385A3.129 3.129 0 0 1 5 8.5C5 6.567 6.79 5 9 5c1.195 0 2.267.458 3 1.185V0h2v6.185C14.733 5.458 15.805 5 17 5c2.21 0 4 1.567 4 3.5a3.13 3.13 0 0 1-.385 1.5H38v2H15.275ZM17 10c1.364 0 2-.914 2-1.5S18.364 7 17 7c-1.364 0-2 .914-2 1.5V10h2Zm-6 0V8.5c0-.586-.636-1.5-2-1.5-1.364 0-2 .914-2 1.5S7.636 10 9 10h2Z" fill="#707070" /></g><defs><clipPath id="pi-gift-card"><path fill="#fff" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```

File: app/assets/images/payment_icons/onlinebanking.svg [hunk 8]
```diff
@@ -0,0 +1 @@
...one" aria-labelledby="pi-onlinebanking"><title id="pi-onlinebanking">Online Banking</title><g clip-path="url(#a)"><rect width="38" height="24" rx="3" fill="#F5F5F5" style="fill:#f5f5f5;fill:color(display-p3 .9608 .9608 .9608);fill-opacity:1"/><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" style="stroke:#000;stroke-opacity:.07"/><path fill-rule="evenodd" clip-rule="evenodd" d="M0 3c0-2 1.5-3 3-3h32c2 0 3 1.5 3 3v1H0V3Zm11-1a1 1 0 1 1-2 0 1 1 0 0 1 2 0ZM7 3a1 1 0 1 0 0-2 1 1 0 0 0 0 2ZM5 2a1 1 0 1 1-2 0 1 1 0 0 1 2 0Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/><path d="M12.557 9.81a.92.92 0 1 0 .724 1.69L18.212 9a2 2 0 0 1 1.576 0l4.93 2.5a.92.92 0 0 0 .725-1.69l-5.655-2.65a2 2 0 0 0-1.576 0l-5.655 2.65Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/><path d="M18 12a1 1 0 1 1 2 0v4a1 1 0 1 1-2 0v-4ZM23 12a1 1 0 0 0-1 1v3a1 1 0 1 0 2 0v-3a1 1 0 0 0-1-1ZM15 12a1 1 0 0 0-1 1v3a1 1 0 1 0 2 0v-3a1 1 0 0 0-1-1ZM13 18a1 1 0 1 0 0 2h12a1 1 0 1 0 0-2H13Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/></g></svg>
```
Comment (jffortier) [inline app/assets/images/payment_icons/onlinebanking.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-onlinebanking"><title id="pi-onlinebanking">Online Banking</title><g clip-path="url(#a)"><rect width="38" height="24" rx="3" fill="#F5F5F5" /><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" /><path fill-rule="evenodd" clip-rule="evenodd" d="M0 3c0-2 1.5-3 3-3h32c2 0 3 1.5 3 3v1H0V3Zm11-1a1 1 0 1 1-2 0 1 1 0 0 1 2 0ZM7 3a1 1 0 1 0 0-2 1 1 0 0 0 0 2ZM5 2a1 1 0 1 1-2 0 1 1 0 0 1 2 0Z" fill="#707070" /><path d="M12.557 9.81a.92.92 0 1 0 .724 1.69L18.212 9a2 2 0 0 1 1.576 0l4.93 2.5a.92.92 0 0 0 .725-1.69l-5.655-2.65a2 2 0 0 0-1.576 0l-5.655 2.65Z" fill="#707070" /><path d="M18 12a1 1 0 1 1 2 0v4a1 1 0 1 1-2 0v-4ZM23 12a1 1 0 0 0-1 1v3a1 1 0 1 0 2 0v-3a1 1 0 0 0-1-1ZM15 12a1 1 0 0 0-1 1v3a1 1 0 1 0 2 0v-3a1 1 0 0 0-1-1ZM13 18a1 1 0 1 0 0 2h12a1 1 0 1 0 0-2H13Z" fill="#707070" /></g></svg>
```

File: app/assets/images/payment_icons/storecredit.svg [hunk 9]
```diff
@@ -0,0 +1 @@
... id="pi-storecredit">Store Credit</title><g clip-path="url(#pi-storecredit)"><rect width="38" height="24" rx="3" fill="#F5F5F5" style="fill:#f5f5f5;fill:color(display-p3 .9608 .9608 .9608);fill-opacity:1"/><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" style="stroke:#000;stroke-opacity:.07"/><path d="m11.469 5.386-.414.532c-.684.878-1.17 2.048-.558 2.977.384.583 1.065 1.098 2.272 1.093 2.77.012 3.185-3.715 3.185-3.715S16.23 10 19 10c2.77 0 3.046-3.727 3.046-3.727s.416 3.727 3.185 3.715c1.207.005 1.888-.51 2.272-1.093.612-.93.126-2.099-.558-2.977l-.414-.532a1 1 0 0 0-.79-.386H12.259a1 1 0 0 0-.789.386ZM13 12v-.017a4.497 4.497 0 0 0 1.87-.477c.083.146.13.314.13.494v5h3v-3a1 1 0 1 1 2 0v3h3v-5c0-.18.047-.348.13-.494a4.497 4.497 0 0 0 1.87.477V17a1 1 0 1 1 0 2H13a1 1 0 1 1 0-2v-5ZM3 11a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-2ZM32 10a1 1 0 0 0-1 1v2a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-2a1 1 0 0 0-1-1h-2Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/></g><defs><clipPath id="pi-storecredit"><path fill="#fff" style="fill:#fff;fill-opacity:1" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```
Comment (jffortier) [inline app/assets/images/payment_icons/storecredit.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-storecredit"><title id="pi-storecredit">Store Credit</title><g clip-path="url(#pi-storecredit)"><rect width="38" height="24" rx="3" fill="#F5F5F5" /><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" /><path d="m11.469 5.386-.414.532c-.684.878-1.17 2.048-.558 2.977.384.583 1.065 1.098 2.272 1.093 2.77.012 3.185-3.715 3.185-3.715S16.23 10 19 10c2.77 0 3.046-3.727 3.046-3.727s.416 3.727 3.185 3.715c1.207.005 1.888-.51 2.272-1.093.612-.93.126-2.099-.558-2.977l-.414-.532a1 1 0 0 0-.79-.386H12.259a1 1 0 0 0-.789.386ZM13 12v-.017a4.497 4.497 0 0 0 1.87-.477c.083.146.13.314.13.494v5h3v-3a1 1 0 1 1 2 0v3h3v-5c0-.18.047-.348.13-.494a4.497 4.497 0 0 0 1.87.477V17a1 1 0 1 1 0 2H13a1 1 0 1 1 0-2v-5ZM3 11a1 1 0 0 1 1-1h2a1 1 0 0 1 1 1v2a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-2ZM32 10a1 1 0 0 0-1 1v2a1 1 0 0 0 1 1h2a1 1 0 0 0 1-1v-2a1 1 0 0 0-1-1h-2Z" fill="#707070" /></g><defs><clipPath id="pi-storecredit"><path fill="#fff" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```

File: app/assets/images/payment_icons/wallet.svg [hunk 10]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-wallet"><title id="pi-wallet">Wallet</title><g clip-path="url(#pi-wallet)"><rect width="38" height="24" rx="3" fill="#F5F5F5" style="fill:#f5f5f5;fill:color(display-p3 .9608 .9608 .9608);fill-opacity:1"/><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" style="stroke:#000;stroke-opacity:.07"/><path d="M4 4a1 1 0 0 0 0 2h30a1 1 0 1 0 0-2H4ZM34 8a1 1 0 1 1 0 2h-4.556c-2.328 0-4.161 2.013-5.758 3.766l-.258.283A5.985 5.985 0 0 1 19 16a5.985 5.985 0 0 1-4.428-1.951l-.258-.283C12.717 12.013 10.884 10 8.556 10H4a1 1 0 1 1 0-2h30Z" fill="#707070" style="fill:#707070;fill:color(display-p3 .4392 .4392 .4392);fill-opacity:1"/></g><defs><clipPath id="pi-wallet"><path fill="#fff" style="fill:#fff;fill-opacity:1" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```
Comment (jffortier) [inline app/assets/images/payment_icons/wallet.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" fill="none" aria-labelledby="pi-wallet"><title id="pi-wallet">Wallet</title><g clip-path="url(#pi-wallet)"><rect width="38" height="24" rx="3" fill="#F5F5F5" /><rect x=".5" y=".5" width="37" height="23" rx="2.5" stroke="#000" stroke-opacity=".07" /><path d="M4 4a1 1 0 0 0 0 2h30a1 1 0 1 0 0-2H4ZM34 8a1 1 0 1 1 0 2h-4.556c-2.328 0-4.161 2.013-5.758 3.766l-.258.283A5.985 5.985 0 0 1 19 16a5.985 5.985 0 0 1-4.428-1.951l-.258-.283C12.717 12.013 10.884 10 8.556 10H4a1 1 0 1 1 0-2h30Z" fill="#707070" /></g><defs><clipPath id="pi-wallet"><path fill="#fff" d="M0 0h38v24H0z"/></clipPath></defs></svg>
```

**Review by jffortier — APPROVED** [review]
:shipit: 

Heads up that if we want to allow theming of these icons by the consumer, we may need to come back to these SVGs and implement some custom properties. All depends of the branding we'll want to do. cc @Jacquesattaque

**Pull request merged**